pkg/cloud/aws/services/ec2/securitygroups.go

```diff
@@ -304,6 +304,13 @@ func (s *Service) getSecurityGroupIngressRules(role v1alpha1.SecurityGroupRole)
 					s.scope.SecurityGroups()[v1alpha1.SecurityGroupNode].ID,
 				},
 			},
+			{
+				Description: "IP-in-IP (calico)",
+				Protocol:    v1alpha1.SecurityGroupProtocolIPinIP,
+				FromPort:    1,
+				ToPort:      65535,
+				CidrBlocks:  []string{anyIPv4CidrBlock},
```

Comment:
Not a nit this time:
We shouldn't use `0.0.0.0/0` here, but instead reference the security group ID using the `SourceSecurityGroupIDs` parameter.